src/resources/i18n/translations/en.json

```diff
@@ -810,5 +810,10 @@
   "Your queue is empty": "Your queue is empty",
   "Your membership will not auto-renew": "Your membership will not auto-renew",
   "You are not being charged during your free trial": "You are not charged during or after the free trial.",
-  "You will not be charged for signing up for your free trial": "You are not charged for signing up or using your free trial. We hope you subscribe (manually) afterwards."
+  "You will not be charged for signing up for your free trial": "You are not charged for signing up or using your free trial. We hope you subscribe (manually) afterwards.",
+  "Android Auto Podcast Title": "Podcast",
+  "Android Auto Queue Title": "Queue",
+  "Android Auto History Title": "History",
+  "Android Auto Permission Title": "Permission Required",
+  "Android Auto Permission Body": "Please enable android auto draw over apps permission."
```

Comment:
This can say something like "Please allow Podverse to be displayed on top when connected to your vehicle." I know its not exactly what it does but I think it gives a less tech-y description to users.

Reply:
up for yall to decide;) im just putting some placeholder texts there to demonstrate functionality